exercises/space-age/README.md

```diff
@@ -83,12 +81,12 @@ stack ghci
 The [exercism/haskell](https://github.com/exercism/haskell) repository on
 GitHub is the home for all of the Haskell exercises.
 
-If you have feedback about an exercise, or want to help implementing a new
-one, head over there and create an issue.  We'll do our best to help you!
+If you have feedback about an exercise or want to help to implement a new one, head over there, and create an issue. We'll do our best to help you!
```

Comment:
This appears to be two changes: An extra comma, and the removal of word wrapping.

I think the comma makes the sentence read less naturally and the removal of word wrapping can be debated, but is probably not warranted on its own.

Reply:
The reasoning that I've added an extra comma was because this sentence was leading into a list. If you'd like, I can revert this as well. The comma before the "and" is debatable.

Reply:
It appears that I misread this change; it is really: removing the comma before "or", adding a comma before "and", and removing word-wrapping. Removing the comma before "or" turns an interposed sentence into a natural extension of the outer sentence.

I don't think the comma before "and" is a list comma. If so, the list has two elements. Typically you don't add [serial comma](https://en.wikipedia.org/wiki/Serial_comma) unless the list has at least three elements. So the overall sentence structure, as I see it (and perhaps this is only perspective) is this: "if (p or q) then (x and y)" with "p = "you have feedback about an exercise", "q = want to help to implement a new one", "x = head over there" and "y = create an issue".